packages/tailscale-pikvm/tailscale-pikvm.install

```diff
@@ -1,8 +1,15 @@
+pre_upgrade() {
+	service_enabled=$(systemctl is-enabled tailscaled)
```

Comment:
make it like this: `$(systemctl is-enabled tailscaled || true)`

Reply:
@mdevaev updated